src/dashboard/src/components/charts/doughnut/totalStorage/TotalStorage.tsx

```diff
@@ -1,23 +1,24 @@
 'use client';
 
+import { convertToStorageSize } from '@/utils';
 import { ArcElement, Chart as ChartJS, Tooltip } from 'chart.js';
 import React from 'react';
 import { Doughnut } from 'react-chartjs-2';
 import styles from './TotalStorage.module.scss';
-import { defaultData } from './defaultData';
+import { useTotalStorageDoughnutChart } from './hooks';
 
 ChartJS.register(ArcElement, Tooltip);
 
 export const TotalStorage: React.FC = () => {
-  const data = defaultData;
+  const data = useTotalStorageDoughnutChart();
```

Comment:
Pull in data from our local storage which is populated when the filter button is clicked.